smoke-tests/src/test/groovy/io/opentelemetry/smoketest/PlaySmokeTest.groovy

```diff
@@ -36,6 +40,9 @@ class PlaySmokeTest extends SmokeTest {
     //Both play and akka-http support produce spans with the same name.
     //One internal, one SERVER
     countSpansByName(traces, '/welcome') == 2
+    def httpRouteAttributes = extractRouteAttributesFrom(traces)
+    httpRouteAttributes.size() == 1
+    httpRouteAttributes.get(0).value.stringValue == "/welcome"
```

Comment:
Nit: you could use `TraceInspector` and `countFilteredAttributes` instead.

Reply:
That's good to know! I simplified that at [5be95b8](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7801/commits/5be95b8fd909eb43935117776cb91819492b9c98).